irepela/homework_9/fraction.py

```diff
@@ -41,43 +41,55 @@ def convert_to_fraction(self, other):
         return result
 
     def __eq__(self, other):
+        result = NotImplemented
         other_fraction = self.convert_to_fraction(other)
         if other_fraction:
```

Comment:
and if other fraction is Fraction(0, 5) it will be false and you return NotImplemented